composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "prooph/pdo-event-sourcing-pack",
+    "description": "A pack to install Prooph PDO event sourcing components in a Symfony Flex application",
+    "type": "symfony-pack",
+    "license": "BSD-3-Clause",
+    "keywords": ["event sourcing", "event store", "cqrs"],
+    "homepage": "http://getprooph.org/",
+    "authors": [
+        {
+            "name": "Gildas Quéméner",
+            "email": "gildas.quemener@gmail.com"
+        }
```

Comment:
What's your policy about the "authors" of Prooph package?

Bundles have:

```yaml
    {
      "name": "Alexander Miertsch",
      "email": "contact@prooph.de",
      "homepage": "http://www.prooph.de"
    },
    {
      "name": "Sascha-Oliver Prolic",
      "email": "saschaprolic@googlemail.com"
    },
    {
      "name": "Malte Blättermann",
      "email": "malte.blaettermann@googlemail.com"
    },
    {
      "name": "Ralf Junghanns",
      "email": "ralf.junghanns@gmail.com"
    },
    {
      "name": "Oskar Pfeifer-Bley",
      "email": "oskar@programming-php.net"
}
```

Should I put the same here @codeliner @UFOMelkor ?

Reply:
I think we can reduce the list to

```json
{
      "name": "Alexander Miertsch",
      "email": "contact@prooph.de",
      "homepage": "http://www.prooph.de"
    },
    {
      "name": "Sascha-Oliver Prolic",
      "email": "saschaprolic@googlemail.com"
    },    
    {
      "name": "Oskar Pfeifer-Bley",
      "email": "oskar@programming-php.net"
}
```